### PR TITLE
Explain Octavia's Prometheus endpoint

### PR DESCRIPTION
**Pull request by fghaas**
* Explain how one adds a metrics endpoint in Octavia (available from OpenStack Antelope forward)
* Add the availability of that feature to the feature support matrix.

### CODE REVIEW

File: docs/howto/openstack/octavia/metrics.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    ```
+
+## Assumptions and scenario
+
+We assume you already have an [HTTPS-terminated load balancer](tls-lb.md) that forwards client requests to a back-end server pool.
+In our test scenario, the load balancer was accepting HTTPS requests for `whoogle.example.com` and forwarding them to a two-member pool, with servers each running a Docker container for [Whoogle Search](https://github.com/benbusby/whoogle-search).
+
+## Creating a metrics listener
+
+Here is `mylb`, the load balancer we used during testing...
```
Comment (colder-is-better) [inline docs/howto/openstack/octavia/metrics.md:37]:
```suggestion
Here is `mylb`, the load balancer we used during testing:
```
(I use the dots when presenting more than one console output box, in the context of a long sentence.)

**Review by colder-is-better — COMMENTED** [review]
Everything looks good, kudos on keeping the Whoogle example, and I only have a sub-atomic nit.

File: docs/howto/openstack/octavia/metrics.md (hunk 1 above)
Comment (fghaas) [reply docs/howto/openstack/octavia/metrics.md]: Point taken (see what I did there?)

**Comment by fghaas** [comment]
I have now reversed the order of the two commits in this branch, and created a separate PR for the Antelope version update (#237).

File: docs/howto/openstack/octavia/metrics.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
++---------------------+--------+
+| Field               | Value  |
++---------------------+--------+
+| provisioning_status | ACTIVE |
++---------------------+--------+
+```
+
+## Testing the listener
+
+Once the listener reports as `ACTIVE`, you should be able to check its endpoint.
```
Comment (colder-is-better) [inline docs/howto/openstack/octavia/metrics.md:117]:
```suggestion
Once the listener reports as being `ACTIVE`, you should be able to check its endpoint.
```

(the above reads better --- at least for me)

**Review by colder-is-better — COMMENTED** [review]
It looks good --- and I have nothing but a small nit.

File: docs/howto/openstack/octavia/metrics.md (hunk 2 above)
Comment (fghaas) [reply docs/howto/openstack/octavia/metrics.md]: Fair point; fixed as suggested.

**Review by noonedeadpunk — APPROVED** [review]
LGTM

**Review by colder-is-better — APPROVED** [review]

**Pull request merged**